### PR TITLE
Added main module.

### PR DESCRIPTION
**Pull request by bujoanna**
**What has changed:**
- Added main menu with three variants.
- Information about the types of tickets;
- Possibility to choose tickets from two available types (normal, reduced).
- Ability to select the number of tickets.
- Possibility to add up selected tickets.

**What reviewer should know:**

An enum named Price,
I once saw code where someone put constant variables with values assigned to it. Not sure if that's the general convention ;)

### CODE REVIEW

File: src/main/java/org/example/ticketOffice/ChooseATicket.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package org.example.ticketOffice;
+
+public interface ChooseATicket {
```
Comment (jacekgolebiewski) [inline src/main/java/org/example/ticketOffice/ChooseATicket.java:3]: warto nazywac klasy rzeczownikami, a metody czasownikami

moze cos w stylu: TicketService, TicketMachine, TicketApp?

**Review by jacekgolebiewski — APPROVED** [review]

File: src/main/java/org/example/ticketOffice/TicketType.java [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package org.example.ticketOffice;
+
+public enum TicketType {
+    NORMALNY,
```
[the remark this answers is not in the document]
Comment (bujoanna) [reply src/main/java/org/example/ticketOffice/TicketType.java:4]: OK

File: src/main/java/org/example/ticketOffice/Price.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
+package org.example.ticketOffice;
+
+enum Price {
```
[the remark this answers is not in the document]
Comment (bujoanna) [reply src/main/java/org/example/ticketOffice/Price.java:3]: Ten twór widziałam kiedyś w necie :) Zmienię. Intuicja mi podpowiadała, że nie jest to standardowe działanie...

File: src/main/java/org/example/ticketOffice/Information.java [hunk 4]
```diff
@@ -0,0 +1,10 @@
+package org.example.ticketOffice;
+
+public class Information {
+
+    public void information() {
```
[the remark this answers is not in the document]
Comment (bujoanna) [reply src/main/java/org/example/ticketOffice/Information.java:5]: Ok :)

File: src/main/java/org/example/ticketOffice/ChooseATicket.java (hunk 1 above)
Comment (bujoanna) [reply src/main/java/org/example/ticketOffice/ChooseATicket.java:3]: Nie wiedziałam. Zmienię.

**Pull request merged**